Jenkinsfile

```diff
@@ -277,6 +290,9 @@ node ('master') {
   cleanWs()
   parallel tasks
 
+  if (build_scenario == 'Before merge to trunk')
+    gitNotify ("New CI: Merge to trunk", "Finish", 'SUCCESS')
+
   // if (currentBuild.currentResult == 'SUCCESS') {
```

Comment:
Quite unrelated to the pr, but still, why is there commented out code?

Reply:
It was for testing purposes, I will delete it.